app/models/goods_nomenclature.rb

```diff
@@ -104,7 +104,8 @@ class GoodsNomenclature < Sequel::Model
                            left_primary_key: :goods_nomenclature_sid,
                            left_key: :goods_nomenclature_sid,
                            right_key: %i[footnote_type footnote_id],
-                           right_primary_key: %i[footnote_type_id footnote_id] do |ds|
+                           right_primary_key: %i[footnote_type_id footnote_id],
```

Comment:
Has this change snuck into this PR? I mean it looks correct to me to order based on the composite primary keys